src/server/components/dashboard/listRouter.ts

```diff
@@ -70,6 +78,8 @@ listRouter.use("/:listId/delete", (req, _res, next) => {
 
 listRouter.get("/:listId/delete", listDeleteController);
 listRouter.post("/:listId/delete", listDeletePostController);
+listRouter.get("/:listId/csv", listExportController);
+listRouter.get("/:listId/csv", validateActionPermission, listExportController);
```

Comment:
should just be one here

Reply:
Must have been an issue with the merge/rebase 🙈